packages/core/src/controllers/relayer.ts

```diff
@@ -474,11 +477,13 @@ export class Relayer extends IRelayer {
   };
 
   private onConnectHandler = () => {
+    this.logger.trace("wss connected");
```

Comment:
Concept of wss vs http as transport is abstracted in the relayer, would be good to avoid references to concrete transport layer:
```suggestion
    this.logger.trace("relayer connected");
```